build_utils.sh

```diff
@@ -11,6 +11,7 @@ function version_tag() {
 function gen_versions()
 {
   local version=$1
+  echo "$version"
```

Comment:
You'll want to change the comment above the method if you're going with this fix. It says this method is only for generating less specific versions.